src/content/updates/roadmap/roadmap.mdx

```diff
@@ -2,9 +2,137 @@
 title: Roadmap
 ---
 
-## Overview
+## Endgame
```

Comment:
Looks good, was just wondering if we should clarify that this is for a minor release? Or maybe explain our release process (i.e. monthly-ish minor releases which have an endgame).

Reply:
Agreed. @joshblack is working on a post explaining our endgame process. I'm switching this to a WIP and will do a bit more cleanup on Monday.